plugins/woocommerce/includes/admin/class-wc-admin-log-table-list.php

```diff
@@ -92,6 +92,38 @@ public function level_dropdown() {
 		<?php
 	}
 
+	/**
+	 * Generates the table rows.
+	 *
+	 * @return void
+	 */
+	public function display_rows() {
+		foreach ( $this->items as $log ) {
+			$this->single_row( $log );
+			$this->context_row( $log );
+		}
+	}
+
+	/**
+	 * Render the additional table row that contains extra log context data.
+	 *
+	 * @param array $log Log entry data.
+	 *
+	 * @return void
+	 */
+	protected function context_row( $log ) {
```

Comment:
Could we bail early here in case there's no additional context? As it is we're always printing these extra rows, regardless of whether there's any context or not.

Reply:
Good point! Updated in 286fb53418846b44a1f69fd8adf67c65024b97b1